pages/landing/components/LandingPageCard.js

```diff
@@ -39,18 +73,41 @@ function LandingPageCard(props) {
               <div className="container">
                 <div>
                   <div className="a-button-container">
-                    <LandingCardButton
-                      classNameProp={props.class1}
-                      name={props.text1}
-                    />
-                    <LandingCardButton
-                      classNameProp={props.class2}
-                      name={props.text2}
-                    />
-                    <LandingCardButton
-                      classNameProp={props.class3}
-                      name={props.text3}
-                    />
+                    {props.fromPage == "request" ? (
+                      <div>
+                        <div className="a-landing-card-button">
+                          <button
+                            className="btn btn-success"
+                            onClick={acceptHandler}
+                          >
+                            Accept
+                          </button>
+                        </div>
+                        <div className="a-landing-card-button">
+                          <button
+                            className="btn btn-danger a-landing-card-button"
+                            onClick={rejectHandler}
```

Comment:
This function don't do anything why are you passing this?

Reply:
Removed sir